frontend/src/common/queries/cellGuide.ts

```diff
@@ -85,12 +87,24 @@ async function fetchQuery({
 /**
  * Generic cell guide hook
  */
-
+// Mapping from organism name to taxon id. Hardcode this for now.
+// This will need to be updated if Census ever includes more organisms.
+export const ORGANISM_NAME_TO_TAXON_ID_MAPPING = {
+  "Homo sapiens": "NCBITaxon_9606",
+  "Mus musculus": "NCBITaxon_10090",
+};
 export function useCellGuideQuery<T = CellGuideResponse>(
```

Comment:
For developer experience and readability, this function has many arguments now, maybe it's time to convert the argument to an object?

E.g.,

```ts
useCellGuideQuery({ dataType, queryId = "", organismName = "", queryLatestSnapshotIdentifier = true}) { ... }
```